### ISSUE TITLE
What does it mean for mi_version==201 && mi_is_redirected==0

### ISSUE THREAD

**Issue by hbsun2113** [microsoft/mimalloc#579]
Hi, @dannal, I print the mi_version and mi_is_redirected in my program, and they return mi_version==201 and mi_is_redirected==0.
According to my knowledge, `mi_version==201` means mimalloc lib is linked successfully, and `mi_is_redirected==0` means malloc is redirected failed(although I have put mimalloc-override.dll in the same folder).
My 2 questions:
1. How can I check whether mimalloc is enabled successfully for my program and DLLs?
2. If redirected failed, how can I make it successful?

**Comment by m-seker** [comment]
Did you also put mimalloc-redirect.dll into the same folder ?

**Comment by hbsun2113** [comment]
yes:
![image](https://user-images.githubusercontent.com/27994442/167524993-6cba0e7d-f6de-4ece-8599-b2726529bfba.png)

**Comment by m-seker** [comment]
mimalloc-redirect.dll and mimalloc-override.dll are not not the same thing.

**Comment by hbsun2113** [comment]
I mean mimalloc-redirect.dll and mimalloc-override.dll are in  the same folder

**Comment by daanx** [comment]
Yes, they should be in the same folder; the version is 2.0.1 which is a bit old -- please try again with the latest v2.0.6.
If `mi_is_redirected() == false` then mimalloc did not redirect the standard malloc. Set the environment variable 
`MIMALLOC_VERBOSE=1` so see any warnings etc. (and use 2 or 3 to see more detailed information whether mimalloc redirection is working or not). Make sure that mimalloc-redirect.lib is as early as possible on the linker command line.
